6._Contributing.md

```diff
@@ -83,3 +85,10 @@ The next section contains more information on the workflow followed for Pull Req
 * Pull requests can be closed by the author without merging.
 
 * Pull requests may be closed by a Maintainer if the decision is made that it is not going to be merged.
+
+**4.6 Blog Post Specific**
+
+* A Community/Official Blog Post is considered to be a broad SLSA announcement that may be in regards to a new release, clarification, new tooling, showcase work, etc.  These blog posts are typically in coordination with the broader SLSA community and comes from full community consensus.  PRs must meet all necessary approvals from at least 2-3 steering committee members from differing organizations.  
+
+* Guest Blog Post are considered to be from a single person/organization that is sharing how they are implementing/applying SLSA in their respective project/organization.  PRs must meet all necessary approvals from at least 2-3 steering committee members, which will assess the accuracy of any claims with regards to SLSA compliance.  _Note: The views expressed by Guests are not official positions of the SLSA community or any parent organization. The author has requested and incorporated reviewer feedback whenever possible, but the opinions presented are the author’s alone_
```

Comment:
Personally I think 1 approval is sufficient for guest posts.

Reply:
Thanks Mark.  I think that was a quote from David W. to have 2-3 steering committee members.  I can change it to 1.